src/schemas/21-526EZ/schema.js

```diff
@@ -376,59 +419,71 @@ let schema = {
       maxItems: 100,
       items: {
         type: 'object',
-        required: ['treatmentCenterName', 'treatmentCenterType'],
-        properties: {
-          treatmentCenterName: {
-            type: 'string',
-            maxLength: 100,
-            pattern: "([a-zA-Z0-9\-'.#]([a-zA-Z0-9\-'.# ])?)+$"
-          },
-          treatmentDateRange: {
-            $ref: '#/definitions/dateRange'
-          },
-          treatmentCenterAddress: {
-            $ref: '#/definitions/vaTreatmentCenterAddress'
-          },
-          treatmentCenterType: {
-            type: 'string',
-            enum: ['VA_MEDICAL_CENTER', 'DOD_MTF']
-          }
-        }
-      }
-    },
-    privateRecordReleases: {
-    // These records are sent through an ancillary form and are not directly
-    // submitted via 526. This ancillary submission process has no actual
-    // validations, but we thought keeping them here (especially for address)
-    // would enforce a baseline of data quality which would be in the
-    // submitter's best interest.
-      type: 'array',
-      items: {
-        type: 'object',
-        required: ['treatmentCenterName'],
+        required: ['center', 'treatmentDateRange'],
         properties: {
-          treatmentCenterName: {
-            type: 'string',
-            maxLength: 100,
-            pattern: "([a-zA-Z0-9\\-'.#]([a-zA-Z0-9\\-'.# ])?)+$"
+          center: {
+            type: 'object',
+            required: ['name', 'type', 'treatmentCenterAddress'],
+            properties: {
+              name: {
+                type: 'string',
+                maxLength: 100,
+                pattern: "^([a-zA-Z0-9\\-'.#]([a-zA-Z0-9\\-'.# ])?)+$"
+              },
+              type: {
+                type: 'string',
+                enum: ['VA_MEDICAL_CENTER', 'DOD_MTF']
+              },
+              treatmentCenterAddress: {
```

Comment:
Should we make a note of all these things that the back end will have to translate from this schema to what EVSS expects?

Reply:
Yeah, that seems pretty important. I'll open an issue for that.